internal/controller/promotionstrategy_controller_test.go

```diff
@@ -1987,6 +1986,21 @@ var _ = Describe("PromotionStrategy Controller", func() {
 				},
 			}
 
+			urlTemplate := `
```

Comment:
Can we go-embed the example YAML and extract the go template? Just trying to avoid maintaining two copies of the test/example template.